src/main/java/de/uhd/ifi/se/decision/management/jira/consistency/LinkSuggestion.java

```diff
@@ -0,0 +1,47 @@
+package de.uhd.ifi.se.decision.management.jira.consistency;
+
+import com.atlassian.jira.issue.Issue;
+
+public class LinkSuggestion {
+
+	public LinkSuggestion(Issue baseIssue, Issue targetIssue, Double score) {
+		this.baseIssue = baseIssue;
+		this.targetIssue = targetIssue;
+		this.score = score;
+	}
+
+	private Issue baseIssue;
+	private Issue targetIssue;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5455522)